css/style.css

```diff
@@ -465,7 +464,7 @@ FOOTER FOOTER
   list-style-type: none;
 }
 
-input {
+.form-name {
   background: #fff;
```

Comment:
- [ ] (OPTIONAL) In order to improve your user interface, it would be good to add some padding to all your inputs. I noted that when filling in the form, my information did not have any padding, so it'll be great if you could address this issue.